src/Data/Foreign.purs

```diff
@@ -81,6 +81,8 @@ renderForeignError (TypeMismatch exp act) = "Type mismatch: expected " <> exp <>
 
 -- | An error monad, used in this library to encode possible failures when
 -- | dealing with foreign data.
+-- | `Except e` was chosen instead of `Either e` because `Except`'s `Alt` instance
```

Comment:
Could you add an extra newline here please, so that this gets rendered as two lines in the output?